patches/chrome-VERSION.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/chrome/VERSION b/chrome/VERSION
-index 3c26e3145cfbeb4928c98357912c90b7b6c4f669..50bdeeab1c597f55ad35713bface77f2a5d832cc 100644
+index 3c26e3145cfbeb4928c98357912c90b7b6c4f669..8e38e4bd7d99ebe81c1b833c50df2dd3bcce0236 100644
 --- a/chrome/VERSION
```

Comment:
This change seems unrelated and unnecessary?

Reply:
Yes, looks like it sneaked in while rebasing/fixing CI. Removed, will keep an eye on the CI build.